compiler/ccgtypes.nim

```diff
@@ -430,9 +430,8 @@ proc genProcParams(m: BModule, t: PType, rettype, params: var Rope,
   else: add(params, ")")
   params = "(" & params
 
-proc mangleRecFieldName(m: BModule; field: PSym, rectype: PType): Rope =
-  if (rectype.sym != nil) and
```

Comment:
semRecordNodeAux propogates importc and exportc object level pragmas to fields anyway